Makefile

```diff
@@ -1,52 +0,0 @@
-GH_PAGES_SOURCES = planet docs Makefile
```

Comment:
At first glance I was concerned by this file removal, however on further review I'm okay with it. There's potential regression in developer-environment functionality **but** most of that is related to the build-and-release process, which shouldn't be occurring for now from this V2 branch anyway. The real reason I'm okay with this is some of those make targets were already fragile/broken so it's a good time to push for a re-do. It's something we will certainly need to resolve before V2 can be released but it doesn't need to happen _here_.

@jreiberkyle before we wrap this PR can you raise an issue to re-implement Makefile functionality for V2 with nox in mind?

Reply:
Great point. Done #262